src/nni_manager/training_service/dlts/dltsTrialConfig.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TrialConfig } from "training_service/common/trialConfig";
+
+export class DLTSTrialConfig extends TrialConfig {
+  public constructor(
+    command: string,
```

Comment:
Unify code style, all fields use `public` statement, or all of them does not use `public`.

Reply:
This is actually [Parameter properties](http://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties)